packages/lesswrong/server/manualMigrations/2025-02-14-updateKarmaNotifDefault.ts

```diff
@@ -0,0 +1,37 @@
+import { registerMigration } from "./migrationUtils";
+import { getSqlClientOrThrow } from "../sql/sqlClient";
+import { karmaChangeNotifierDefaultSettings } from "@/lib/collections/users/schema";
+import Users from "@/lib/collections/users/collection";
+
+/**
+ * For A/B testing: When running locally this will be set to a control group of 50
+ * moderately active users.
+ *
+ * TODO generate list
+ */
+const USERS_MANUALLY_EXCLUDED: string[] = [];
```

Comment:
I'm planning to generate a random list of ~50 users in the top 10% of karma recipients, to test whether this results in them posting more in the following month